jupyterlab_gitsync/src/service/file.ts

```diff
@@ -23,6 +23,10 @@ export class File implements IFile {
     right: number,
     bottom: number 
   }
+  cursor: {
+    line: number,
+    ch: number
```

Comment:
you used line and column in notebook file. Let's use consistent names.

Reply:
I use `ch` and `column` because for one I use the `CodeMirror` API directly whereas the other I use the `CodeMirrorEditor` wrapper. I can see about only using the CodeMirror API in the notebook files in a separate PR.